lib/quadquizaminos/contests.ex

```diff
@@ -244,6 +244,25 @@ defmodule Quadquizaminos.Contests do
     RSVP.changeset(rsvp, attrs, %User{})
   end
 
+  @doc """
+  deletes RSVP from the database
+  """
+  @spec cancel_rsvp(integer(), User.t()) :: {:ok, RSVP.t()} | {:error, Changeset.t()}
+  def cancel_rsvp(contest_id, %User{} = user) do
+    contest_id
+    |> get_contest()
```

Comment:
Why do we need to get the contest first?

Reply:
I needed to use the contest struct.
`user_contest_rsvp_query/2` takes in the user struct and contest struct.